charts/ext-postgres-operator/Chart.yaml

```
@@ -8,5 +8,5 @@ description: |
 
 type: application
 
-version: 2.0.1
+version: 2.1.1
```

```suggestion
version: 2.1.0
```

Can you change the version to 2.1.0 ?